src/code_f2470_len_27f0.c

```diff
@@ -191,26 +191,145 @@ INCLUDE_API_ASM(code_f2470_len_27f0, SetNpcFlagBits);
 
 INCLUDE_API_ASM(code_f2470_len_27f0, GetNpcPos);
 
-INCLUDE_API_ASM(code_f2470_len_27f0, EnableNpcShadow);
+ApiStatus EnableNpcShadow(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* ptrReadPos = script->ptrReadPos;
+    NpcId npcID = get_variable(script, *ptrReadPos++);
+    s32 enableShadow = get_variable(script, *ptrReadPos++);
+    Npc* npc = resolve_npc(script, npcID);
+
+    if (npc != NULL) {
+        if (enableShadow) {
+            enable_npc_shadow(npc);
+            return ApiStatus_DONE2;
+        }
+        else {
```

Comment:
styling here, and maybe removal of the else entirely (as below with EnableNpcBlur)